### PR TITLE
hcp/openstack: periodic jobs for Cinder & Manila CSI

### PR DESCRIPTION
**Pull request by EmilienM**
Add periodic jobs for Hypershift on OpenStack with Cinder and Manila
CSI.

### CODE REVIEW

**Comment by EmilienM** [comment]
/cc stephenfin MaysaMacedo mandre

FYI about this PR.

**Comment by EmilienM** [comment]
/hold
I want to make sure things are stabilized, we still have a couple of patches to land and I want to see how it works here before anything.

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-openstack-csi-cinder

**Comment by MaysaMacedo** [comment]
/lgtm
/hold

depends on the backport of https://github.com/openshift/hypershift/pull/5178

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-csi-cinder periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-csi-manila

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-csi-cinder periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-csi-manila

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-csi-manila periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-openstack-csi-manila periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-openstack-csi-manila

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-csi-manila periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-openstack-csi-manila periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-csi-cinder periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-openstack-csi-cinder

**Comment by EmilienM** [comment]
We need https://github.com/openshift/hypershift/pull/5198 to be landed first.

**Comment by EmilienM** [comment]
I'm waiting on a new release payload on 4.18 including the CIRO change in HCP before rehearsing the 4.18 jobs here.

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-openstack-csi-cinder periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-openstack-csi-manila

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.18__periodics.yaml [hunk 1]
```diff
@@ -207,6 +207,34 @@ tests:
     - env: OPENSTACK_CLOUD
       resource_type: openstack-vexxhost-quota-slice
     workflow: hypershift-openstack-conformance
+- as: e2e-openstack-csi-cinder
+  minimum_interval: 72h
+  steps:
+    cluster_profile: hypershift
+    env:
+      CLUSTER_TYPE_OVERRIDE: openstack-vh-mecha-central
```
Comment (mandre) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.18__periodics.yaml:215]: do they need to run on mecha-central? Other environments might be less loaded.

Comment (EmilienM) [reply ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.18__periodics.yaml:215]: yes, CSI tests don't run fine on VEXXHOST yet. We'll take care of that.

**Comment by EmilienM** [comment]
We'll probably need to rebase it on top of https://github.com/openshift/release/pull/59598/

**Comment by EmilienM** [comment]
/hold
We'll probably need to rebase it on top of https://github.com/openshift/release/pull/59598

**Comment by EmilienM** [comment]
/hold cancel
the rename of the job won't land today so I'm trying to get things done, let's merge this one first. I'll update the rename PR.

**Pull request merged**